src/frameworks/drupal8/solr.md

```diff
@@ -14,17 +53,38 @@ And then commit the changes to `composer.json` and `composer.lock`.
 
 ## Configuration
 
-Go to `/admin/config/search/search_api` and click on *Add Server*. Give
-the server a name and a description.
+The configuration can be managed from `settings.platformsh.php` by adding this code snippet:
```

Comment:
Let's put information up here in the intro about the parts that are worth editing.  (The relationship variable and the Drupal server name.)